pypeit/spectrographs/p200_ngps.py

```diff
@@ -0,0 +1,619 @@
+"""
+Module for P200/NGPS specific methods.
+
+.. include:: ../include/links.rst
+"""
+from typing import List, Optional
+
+import numpy as np
+
+from astropy.io import fits
+from astropy.coordinates import Angle
+from astropy import units as u
+from astropy.time import Time
+
+from pypeit import msgs
+from pypeit import io
+from pypeit import telescopes
+from pypeit.core import framematch
+from pypeit.spectrographs import spectrograph
+from pypeit.core import parse
+from pypeit.images import detector_container
+
+
+def flip_fits_slice(s: str) -> str:
```

Comment:
Can you put this into `pypeit/core/parse.py` and update both this file and `p200_dbsp.py` to use one function?

Reply:
Thanks!